### ISSUE TITLE
Newly created MovieClip (defined by AnimateCC script) is unstoppable

### ISSUE THREAD

**Issue by tjutasi** [CreateJS/EaselJS#779]
Since AnimateCC publish everything as MovieClip the newly created MovieClips is unstoppable right after the creation, could stop only some time later. (FlashCC published the same objects as Container, and there was no problem to stop them) for example:

**this.BG_holder=new lib.abui_BackGround();
this.BG_holder.title.gotoAndStop(0);**

and **this.BG_holder.title** still loops it's timeline. When I click a button which call **this.BG_holder.title.gotoAndStop(0)** it's stopped at frame 0.

We can't use AnimateCC because of this bug.

**Comment by ggVGc** [comment]
Yeah, I am pretty sure this is a bug. I sent a PR regarding it a few days ago, https://github.com/CreateJS/EaselJS/pull/778

**Comment by lannymcnie** [comment]
We are aware of this bug, and are hoping to have a solution for it in a future release. In the mean time, I believe forcing an update on the parent timeline can resolve this:

```js
this.BG_holder=new lib.abui_BackGround();
this.BG_holder.gotoAndStop(0); // Force update
this.BG_holder.title.gotoAndStop(0);
```

**Comment by tjutasi** [comment]
I don't see what **this.BG_holder.title** should be here, and there is no 'parent timeline'.

**Comment by lannymcnie** [comment]
This is based on your code. Basically, you have to force `this.BG_holder` (the parent timeline of the "title" instance) to update first, and then you can call `title.gotoAndStop(0)`.

**Comment by Trost123** [comment]
+1 to this.
Is there a chance to bring Container class back to Animate CC, so we can get rid of this bug?

**Comment by lannymcnie** [comment]
That is the hope. This was a change by Adobe, so we are working with them to resolve. Cheers.
